simulation_parameters/microbe_stage/biomes.json

```diff
@@ -1559,6 +1568,7 @@
                 },
                 "iceShard": {
                     "name": "ICESHARD",
+                    "identifier": "ice",
```

Comment:
maybe "damageType" would be a better name here?

Reply:
Sounds good.